### DISCUSSION TITLE
if element‘s text has symbol '&'，the String after symbol '&' will be cut

### DISCUSSION THREAD

**Discussion by MyPainkiller** [drmohundro/SWXMLHash#239] [Q&A]
for example,the element's text is ‘http://99.12.73.74/shorturl/ivnfR2?data=xxxx&ET=B’, it will be parsed into 'http://99.12.73.74/shorturl/ivnfR2?data=xxxx'

**Comment by drmohundro** [comment]
Hi @MyPainkiller, that is expected behavior because `&` is an escape character in XML. If you want the `&` to show up, you'd need it to look like `&amp;` or `&#038;`. Let me know if I can clarify.
